openhtf/core/phase_executor.py

```diff
@@ -91,12 +91,13 @@ class PhaseExecutionOutcome(collections.namedtuple(
   other value will raise an InvalidPhaseResultError.
   """
 
-  def __init__(self, phase_result):
+  def __new__(cls, phase_result):
```

Comment:
why is this changed? We used `__init__` because we're not modifying anything, just checking that the arguments are valid

Reply:
I see this:
    self = super(PhaseExecutionOutcome, cls).__new__(cls, phase_result)
  File "<string>", line 14, in __new__
TypeError: tuple.__new__(X): X is not a type object (PhaseExecutionOutcome)

Reply:
https://stackoverflow.com/questions/42385916/inheriting-from-a-namedtuple-base-class-python